### DISCUSSION TITLE
🎉 New Feature: Right-click to edit (v1.5.20)

### DISCUSSION THREAD

**Discussion by chrisgurney** [chrisgurney/obsidian-note-toolbar#43] [Announcements]
# New Feature 🎉

Right-clicking on the toolbar now lets you edit the toolbar directly (and see which one it is), OR go to the settings! _Super convenient!_

![Right-click menu showing options for; Edit TOOLBARNAME and Note Toolbar Settings](https://github.com/chrisgurney/obsidian-note-toolbar/assets/17337703/4e00c7e5-12a7-4992-9f6b-5762030fa250)

Note: The menu's icons will not appear unless _Obsidian Settings > Appearance > Native menus_ is disabled.

**Comment by FeralFlora** [comment]
This is very convenient, thanks!
